packages/components/src/Button/button.stories.tsx

```diff
@@ -41,123 +42,10 @@ const Template: Story<Args> = (args) => <Button {...args} />;
 export const Primary = Template.bind(null);
 Primary.args = {
   children: "Button",
-  variant: "flat",
 };
 
-export const Secondary = Template.bind(null);
-Secondary.args = {
-  children: "Button",
-  variant: "outline",
-};
-
-export const Tertiary = Template.bind(null);
-Tertiary.args = {
-  children: "Button",
-  color: "neutral",
-  variant: "outline",
-};
-
-export const Destructive = Template.bind(null);
-Destructive.args = {
-  children: "Button",
-  color: "alert",
-  variant: "flat",
-};
-
-export const PrimaryMedium = Template.bind(null);
-PrimaryMedium.args = {
-  children: "Button",
-  size: "medium",
-  variant: "flat",
-};
-
-export const SecondaryMedium = Template.bind(null);
-SecondaryMedium.args = {
-  children: "Button",
-  size: "medium",
-  variant: "outline",
-};
-
-export const TertiaryMedium = Template.bind(null);
-TertiaryMedium.args = {
-  children: "Button",
-  color: "neutral",
-  size: "medium",
-  variant: "outline",
-};
-
-export const Link = Template.bind(null);
-Link.args = {
-  children: "Link Button",
-  variant: "link",
-};
-
-export const linkInBody: Story<Args> = (args) => (
-  <Text size="body">
-    This text surrounds the <Button {...args} /> and shows that the link should
-    adhere to its appearance
-  </Text>
-);
-linkInBody.args = {
-  children: "Link Button",
-  color: "brand",
-  variant: "link",
-};
-
-export const linkInHeading: Story<Args> = (args) => (
-  <Text size="h1">
-    This text surrounds the <Button {...args} /> and shows that the link should
-    adhere to its appearance
-  </Text>
-);
-linkInHeading.args = {
-  children: "Link Button",
-  variant: "link",
-  color: "brand",
-};
-
-export const plainMedium = Template.bind(null);
-plainMedium.args = {
-  children: (
-    <>
-      Button{" "}
-      <AddRoundedIcon className={styles.iconButton} purpose="decorative" />
-    </>
-  ),
-  color: "brand",
-  size: "medium",
-  variant: "plain",
-};
-
-export const linkWithIcon = Template.bind(null);
-linkWithIcon.args = {
-  children: (
-    <>
-      Link with icon{" "}
-      <AddRoundedIcon className={styles.iconLink} purpose="decorative" />
-    </>
-  ),
-  color: "success",
-  variant: "link",
-};
-
-export const outlineWithIcon = Template.bind(null);
-outlineWithIcon.args = {
-  children: (
-    <>
-      Outline with icon{" "}
-      <AddRoundedIcon className={styles.iconButton} purpose="decorative" />
-    </>
-  ),
-  color: "warning",
-  variant: "outline",
-};
-
-export const plainWithOnlyIcon = Template.bind(null);
-plainWithOnlyIcon.args = {
-  children: <AddRoundedIcon purpose="informative" title="add" />,
-  variant: "plain",
-};
+export const RecommendedVariants = () =>
```

Comment:
Are we basically trading all the individual stories for 1 or 2 bigger ones that show more examples together?

Edit: found where you said:

> My reason for putting all these test cases in a single story, instead of having a bunch of individual ones, is it can go in zeroheight as one chunk. But I'm open to revisiting that if people feel it's better to have individual stories.

I'm not sure what the right answer is. Seems like there's tradeoffs either way.

One big one loses searching for or linking to a specific story. But makes it easier load into Zeroheight it seems.

Reply:
One possibility could be to do both. Individual stories could also be re-used in a larger story.

Not sure if that is too much work or not, though.

Reply:
Do you mean like have x individual stories and then one repeater that houses all of them?

Reply:
I'm personally not too opinionated about which direction we choose to go in. Do you have thoughts @anniehu4 (since you're been owning the zeroheight changes)?

Reply:
Yeah, meant X individual stories, and then the big ones for zeroheight that re-use them.

So like

```jsx
export const Modelo = () => {...}

export const Pacifico = ( ) => {...}

export const AllTheCerveza = () => (
  <ul>
    <li>{Modelo()}</li>
    <li>{Pacifico()}</li>
  </ul>
);
```

Just an idea. Not sure if this makes the problem worse or better

Reply:
Maybe we could split so each Heading level in the recommended variants is its own story? then
- RecommendedVariants could compose all of them like Andrew suggested
- we could rename that first story "Default"

Reply:
Sounds good! 👍